src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -460,14 +460,12 @@ private void setupTalons() {
 
     this.setTalonMode(NeutralMode.Brake);
 
-    // Set the sensor phase of the master talons
-    if (RobotBase.isSimulation()) {
-      // TODO: Understand why the sensor phase needs to be swapped for simulation
-      rightMaster.setSensorPhase(false);
-      leftMaster.setSensorPhase(false);
-    } else {
+    if (Robot.checkCapability("hasInvertedDriveEncoderSensorPhase")) {
       rightMaster.setSensorPhase(true);
       leftMaster.setSensorPhase(true);
+    } else {
+      rightMaster.setSensorPhase(false);
+      leftMaster.setSensorPhase(false);
```

Comment:
Do we know why the sensor phase needs to be changed?

Reply:
My understanding is the kitbot drivetrain doesn't have a fixed "front" vs "back", so the encoder phase depends on how they are mounted and what we decide "forward" on the robot is.  According to the docs, the phase needs to be determined empirically by driving the robot forward and set the phase such that the both left and right encoder values are positive.